### PR TITLE
HTML Main Menu

### PR DESCRIPTION
**Pull request by GuestSneezeOSDev**

### CODE REVIEW

**Comment by GuestSneezeOSDev** [comment]
@andrewmcwatters @thepixelmoon could you review this?

**Comment by ThePixelMoon** [comment]
No.

**Comment by GuestSneezeOSDev** [comment]
> No.

Why not?

File: mp/game/hl2sb/gameinfo.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
 "GameInfo"
 {
-	game	"My First HL2MP Mod"
-	title	"HALF+LIFE'"
-	title2	"deathmatch mod"
+	game	"Half-Life 2 Sandbox"
+	title	" "
+	//title2	""
```
Comment (andrewmcwatters) [inline mp/game/hl2sb/gameinfo.txt:5]: OK, this is ready to be promoted to `master`, but could you `git rebase` and drop the changes from `gameinfo.txt`, please?

Thank you!

**Comment by andrewmcwatters** [comment]
Great work, @GuestSneezeOSDev !

**Comment by andrewmcwatters** [comment]
###### Reviewers' notes
- [ ] `git commit --author="YourLocalMoon yourlocalmoon7@gmail.com"`

File: mp/game/hl2sb/gameinfo.txt (hunk 1 above)
Comment (GuestSneezeOSDev) [reply mp/game/hl2sb/gameinfo.txt]: https://github.com/Planimeter/hl2sb-2013/pull/2/commits/a4748dac1121ec1bfe93fe6e570172971edf364f

**Pull request closed without merging**